multiwallet.go

```diff
@@ -37,13 +37,25 @@ type MultiWallet struct {
 }
 
 func NewMultiWallet(rootDir, dbDriver, netType string) (*MultiWallet, error) {
-	rootDir = filepath.Join(rootDir, netType)
-	initLogRotator(filepath.Join(rootDir, logFileName))
 	errors.Separator = ":: "
 
+	if rootDir == "" {
+		return nil, errors.E(errors.Invalid, "rootDir must be a valid folder path")
+	}
+
+	if netType == "" {
+		netType = chaincfg.MainNetParams().Name
+	}
+
 	chainParams := utils.ChainParams(netType)
 	if chainParams == nil {
-		return nil, errors.E("unsupported network type: %s", netType)
+		return nil, errors.Errorf("unsupported network type: %s", netType)
```

Comment:
what is wrong with `fmt.Errorf`?

Reply:
github.com/decred/dcrwallet/errors is used all through the project